core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -9526,8 +9563,11 @@ private void checkLiteral2(String expression, String expected) {
         + "from \"foodmart\".\"reserve_employee\"";
     final String expectedStarRocks = "SELECT TRIM(' str ')\n"
         + "FROM `foodmart`.`reserve_employee`";
+    final String expectedSqlite = "SELECT TRIM(' str ')\n"
+        + "FROM \"foodmart\".\"reserve_employee\"";
     sql(query).withStarRocks().ok(expectedStarRocks)
-        .withDoris().ok(expectedStarRocks);
+        .withDoris().ok(expectedStarRocks)
```

Comment:
Don't add to the end. Add in alphabetical order.

This leads to fewer lines changed, less chance for conflict.